Apps/WebClient/src/ClientApp/src/components/credential/credentialManagement.vue

```diff
@@ -29,6 +37,21 @@ export default class CredentialManagementView extends Vue {
         <page-title title="Credentials" />
         <credential-collection-card />
         <credential-list v-if="isConnectionConnected" class="mt-3" />
+        <b-row v-if="isConnectionConnected" class="mt-3"
```

Comment:
Don't we want to hide it until they have at least 1 credential in wallet?

Reply:
The story does not have this as a requirement :)